kbase-extension/static/kbase/js/widgets/appWidgets2/validators/resolver.js

```diff
@@ -15,22 +15,13 @@ define(['require', 'bluebird'], (require, Promise) => {
         dynamicDropdown: 'dynamicDropdown',
     };
 
-    function getValidatorModule(fieldSpec) {
-        const moduleName = typeToValidatorModule[fieldSpec.data.type];
-        if (!moduleName) {
-            throw new Error('No validator for type: ' + fieldSpec.data.type);
-        }
-        return moduleName;
-    }
-
     function validate(fieldValue, fieldSpec) {
         return new Promise((resolve, reject) => {
-            try {
-                var validatorModule = getValidatorModule(fieldSpec);
-            } catch (ex) {
-                reject(ex);
+            const fieldType = fieldSpec.data.type;
```

Comment:
A little style tweaking and consolidation of an extra (imo unnecessary) function.